js/imageeditor/shared/core/editor.ts

```diff
@@ -295,20 +296,24 @@ export class LayerManager {
 		width,
 		height,
 		layer_name,
-		user_created
+		user_created,
+		layer_id = undefined,
```

Comment:
If we are 'refreshing' layers because the image size has changed, ideally we don't create a new ID, that way we can track active layers more easily.